docs/src/user/settings.md

```diff
@@ -0,0 +1,14 @@
+# Settings
+
+Ambient supports a number of settings that can be configured using the `settings.toml` file. This file is located under the platform's config directory, e.g. `C:\Users\*USER*\AppData\Roaming\Ambient\Ambient\config\settings.toml` on Windows.
+
+## Settings
+
+[general]\
```

Comment:
You can use three-backticks followed by toml here, like here https://github.com/AmbientRun/Ambient/blob/main/docs/src/reference/models.md?plain=1#L8-L12